### PR TITLE
 Add feature status page for Service Credential Distribution [Stable]

### PR DESCRIPTION
**Pull request by johnma14**
@liminw Can you please take a look at this feature status when you get some time. This feature has been marked Stable according to https://istio.io/latest/about/feature-stages/#security-and-policy-enforcement. We need to go through the checklist for Stable and mark those that apply and any additional design docs related to this feature. Appreciate your help.

### CODE REVIEW

File: features/k8s-service-credential-distribution.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+### Requirements: 
+
+**Performance**
+
+- [ ] Latency, throughput, and scalability are quantified and documented on
+	istio.io. 
+
+**Bugs**
+
+- [ ] Feature has no known major issues.
```
Comment (liminw) [inline features/k8s-service-credential-distribution.md:207]: Check.

File: features/k8s-service-credential-distribution.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+- [ ] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+---
+
+## Stable
+
+### Requirements: 
+
+**Performance**
+
+- [ ] Latency, throughput, and scalability are quantified and documented on
```
Comment (liminw) [inline features/k8s-service-credential-distribution.md:202]: Check. https://istio.io/latest/docs/ops/deployment/performance-and-scalability/

File: features/k8s-service-credential-distribution.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+**Bugs**
+
+- [ ] Feature has no known major issues. 
+
+**Approvals**
+
+- [ ] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [ ] The [supportability review panel](https://docs.google.com/document/d/1w0epyFhhDSf_TwFEfa_lrn1v61mXNJKpEp_kUgp4sSc/edit#) has reviewed the feature in order to find any supportability concerns.  
+- [ ] The TOC has reviewed and approved promotion of the feature as part of the
```
Comment (liminw) [inline features/k8s-service-credential-distribution.md:213]: Check for 1 and 3. Do we need supportability review for existing stable features?

File: features/k8s-service-credential-distribution.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+	roadmap for a release.
+
+
+**Promotion**
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [ ] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
```
Comment (liminw) [inline features/k8s-service-credential-distribution.md:221]: Yes. Already marked as stable.

**Comment by johnma14** [comment]
Thank you so much @liminw for taking a look at this PR and adding your thoughts. I have incorporated the changes. Appreciate the help.

**Pull request merged**